mineral-ui/mineral-ui
# [website] Normalize AT usage & p tags in Callout; Fix internal links

### Description

* Normalize all references involving AT to 'assistive technologies'
* Normalize `<p>` tags for instances of Callout & update `<p>` styling in Callout
* Update all internal links to correctly route to components
* Change all relative links to be relative-to-the-root (as should've been done in #612)

### Motivation and context

Closes #632 
Closes #633 

### Screenshots, videos, or demo, if appropriate

Then: 
![screen shot 2018-03-19 at 5 46 46 pm](https://user-images.githubusercontent.com/10079657/37628126-aa2e51dc-2b9d-11e8-81bb-65720da73c3a.png)
![screen shot 2018-03-19 at 5 20 44 pm](https://user-images.githubusercontent.com/10079657/37628146-c16f9d74-2b9d-11e8-97cf-676c9758fcbd.png)

Now: 
![screen shot 2018-03-19 at 5 48 13 pm](https://user-images.githubusercontent.com/10079657/37628148-c9ace7bc-2b9d-11e8-8dc0-fad0cd1fd012.png)
![screen shot 2018-03-19 at 5 53 47 pm](https://user-images.githubusercontent.com/10079657/37628311-8939398c-2b9e-11e8-93b6-31dcb3cdc120.png)


https://632-normalization--mineral-ui.netlify.com/

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to `Getting Started` page
3. Confirm that the callout content is wrapped in a paragraph
4. Repeat steps 2. & 3. for all components
5. Click internal links to ensure you are routed to `mineral-ui.com/components/<component-name>` and not `mineral-ui.com/components/<component-name>` (e.g. the first `Icons` link on http://mineral-ui.com/components/avatar)

### Types of changes

- Bug fix (non-breaking change which fixes an issue)
- Other (provide details below)

Website stuff

### Checklist




### How does this PR make you feel?

![Paragraph dropping](https://media.giphy.com/media/148i4W2ED3mbtK/giphy.gif)
